superset-frontend/plugins/plugin-chart-echarts/src/Timeseries/transformProps.ts

```diff
@@ -227,30 +230,43 @@ export default function transformProps(
     contributionMode || isAreaExpand ? ',.0%' : yAxisFormat,
   );
 
+  const array = ensureIsArray(chartProps.rawFormData?.time_compare);
+  const inverted = invert(verboseMap);
+
   rawSeries.forEach(entry => {
     const lineStyle = isDerivedSeries(entry, chartProps.rawFormData)
       ? { type: 'dashed' as ZRLineType }
       : {};
-    const transformedSeries = transformSeries(entry, colorScale, {
```

Comment:
A thought for future improvement - we may want to use different patterns to disambiguate the different derived series from one another. dasharray could be useful to create half a dozen or so different patterns:
- https://echarts.apache.org/en/option.html#legend.lineStyle.type
- https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/stroke-dasharray

Reply:
@villebro we did a quick hackathon on this, and the echart option works great. We haven't put up a PR yet because there's more that needs to be done around toggling the visualization on/off per user.

Reply:
@eschutho nice, this would be a really nice improvement! ❤️